src/custom-types.json

```diff
@@ -78,7 +78,11 @@
   },
   "application/javascript": {
     "charset": "UTF-8",
-    "compressible": true
+    "compressible": true,
+    "extensions": ["mjs"],
+    "sources": [
+      "https://github.com/nodejs/node-eps/blob/master/002-es-modules.md"
```

Comment:
Yea, unless it comes from one of our sources, it's standard procedure to collect this before adding to custom. It seems to have been useful, as it sounds like the act of clarifying may have changed what it is going to be. If it does end up landing as `text/javascript`, please feel free to update the PR and I can merge 👍